Sources/DeepLinking/Router.swift

```diff
@@ -9,7 +9,9 @@
 import UIKit
 
 public protocol RouteHandler {
-    func handle(route: URL, parameters: [String : String], queryItems: [URLQueryItem])
+    associatedtype T
+
+    func handle(route: URL, parameters: [String : String], queryItems: [URLQueryItem], completion: ((T) -> Void)?)
```

Comment:
Ah thats my bad. Sorry I didn't notice it was a protocol. I just saw it in the diff and assumed it was a class 😞